tsconfig.spec.json

```diff
@@ -1,20 +1,38 @@
 {
-	"extends": "./tsconfig.json",
+	"extends": "./.svelte-kit/tsconfig.json",
```

Comment:
can you please provide configuration - `tsconfig` and vite config -changes in a separate PR.

Reply:
i actually need it for the test, otherwise it would not run correctly